File: Front End/src/app/Core/models/UserVM.ts
import {AppUser} from '../../Admin/Models/AppUser';
import {PrivateCustomer} from '../../Admin/Models/PrivateCustomer';
import {BusinessCustomer} from '../../Admin/Models/BusinessCustomer';

export class UserVM{
  user:AppUser
  privateCustomer:PrivateCustomer
  businessCustomer:BusinessCustomer
}

File: Back End/CamexAPI/Models/VM/UserVm.cs
using Entity.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CamexAPI.Models.VM
{
    public class UserVm
    {
        public AppUser User { get; set; }
        public BusinessCustomer BusinessCustomer { get; set; }
        public PrivateCustomer PrivateCustomer { get; set; }
    }
}
